src/api/spec/models/obs_factory/staging_project_spec.rb

```diff
@@ -379,6 +379,26 @@
 
       it { expect(subject.build_state).to eq(:acceptable) }
     end
+
+    context 'with repositories' do
+      let!(:standard_repo) { create(:repository, project: staging_a) }
+
+      it 'reports building with disabled project' do
+        allow_any_instance_of(ObsFactory::StagingProject).to receive(:building_repositories).and_return([])
+        # overall state is not acceptable if nothing built
+        staging_a.flags.create(flag: 'build', status: 'disable')
```

Comment:
Those three lines should be in a `before` block. And please use a factory for creating the flag: `create(:build_flag, status: 'disable', project: staging_a)`